README.md

```diff
@@ -19,8 +19,19 @@ Prerequisites for local run:
 
 **2. Test run - main notes.**
 
-Upon cloning the repo and trying to run tests, you may be prompted to install the browsers:
-`npx playwright install`
+Once the repo is cloned you can follow these steps to install node, the project dependencies, and Playwright with the latests browsers.
```

Comment:
The node was installed before, better to say:
_Once the repo is cloned you can follow these steps to use the correct node version, install the project dependencies..._

Reply:
Ok, I find it right. thanks